frontend/app/configuration/Config.scala

```diff
@@ -5,7 +5,6 @@ import com.amazonaws.auth.{AWSCredentialsProviderChain, InstanceProfileCredentia
 import com.gu.googleauth.{GoogleAuthConfig, GoogleServiceAccount}
 import com.gu.identity.cookie.{PreProductionKeys, ProductionKeys}
 import com.gu.membership.salesforce.Tier
-import com.gu.membership.touchpoint.TouchpointBackendConfig
```

Comment:
This file only seems to have import changes! This PR is pretty big - try to exclude unnecessary changes if possible.